lib/CMakeLists.txt

```diff
@@ -200,6 +207,11 @@ if(QUDA_VERBOSE_BUILD)
   target_compile_options(quda PRIVATE $<$<COMPILE_LANG_AND_ID:CUDA,NVIDIA>:--ptxas-options=-v>)
 endif(QUDA_VERBOSE_BUILD)
 
+if (${CMAKE_CUDA_COMPILER_ID} MATCHES "NVHPC" AND NOT ${CMAKE_BUILD_TYPE} MATCHES "DEBUG")
```

Comment:
this if statement partially replicates the generator expression. Seems unnecessary to duplicate this.

Reply:
Indeed, redundant, but I don't think this is a big deal.